internals/cli/cmd_run.go

```diff
@@ -275,3 +284,22 @@ func convertArgs(args [][]string) (map[string][]string, error) {
 
 	return mappedArgs, nil
 }
+
+// getAdditionalAdminUIDs parses additional admin user ids passed in via the PEBBLE_ADMINS
+// environment variable.
+func getAdditionalAdminUIDs() ([]sys.UserID, error) {
+	envValue := os.Getenv("PEBBLE_ADMINS")
+	if envValue == "" {
+		return nil, nil
+	}
+	uidStrings := strings.Split(envValue, ",")
+	uids := make([]sys.UserID, 0, len(uidStrings))
+	for _, uidStr := range uidStrings {
+		uid64, err := strconv.ParseUint(uidStr, 10, 32)
+		if err != nil {
+			return nil, fmt.Errorf("cannot parse value %q in $PEBBLE_ADMINS: %w", uidStr, err)
```

Comment:
I think this error message ends up a bit messy, with the `strconv.ParseUint` implementation details in there. What about dropping the `err` here and doing:

```go
fmt.Errorf("cannot parse PEBBLE_ADMINS: %q is not a valid user ID", uidStr)
```